tests/cmd/generate/oci_image/test_module_init.py

```diff
@@ -327,58 +343,6 @@ async def test_GenerateOciImageCommand__handle_bom_inputs(
             mock_load_sbom.assert_not_awaited()
 
 
-@pytest.mark.asyncio
-@patch("mobster.cmd.generate.oci_image.download_parent_image_sbom")
-async def test_GenerateOciImageCommand__execute_contextual_workflow_parent_is_ctx(
-    mock_download_parent: AsyncMock,
-    spdx_parent_sbom_contextualized_json: dict[str, Any],
-    spdx_component_sbom: Document,
-) -> None:
-    """
-    Testcases will be expanded when matching mechanism will be implemented in ISV-
-    Currently is matching mechanism dummy and matches only SPDX ID-s
-    (what will not be a case in full implementation)
-    """
-    mock_download_parent.return_value = spdx_parent_sbom_contextualized_json
-
-    command = GenerateOciImageCommand(MagicMock())
-    contextual_sbom = await command._execute_contextual_workflow(
-        deepcopy(spdx_component_sbom),
-        Image("registry.access.redhat.com/ubi8/ubi", "sha256:sha"),
-        "amd64",
-    )
-    with open(
-        "tests/sbom/test_oci_generate_data/contextual/output/expected_out_component_parent_contextualized.spdx.json",
```

Comment:
This data appears to be unused now that the test referencing it is deleted. We should consider also deleting the file.